staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -433,7 +433,7 @@ func NewCacherFromConfig(config Config) (*Cacher, error) {
 				if !cacher.isStopped() {
 					cacher.startCaching(stopCh)
 				}
-			}, time.Second, stopCh,
+			}, time.Nanosecond, stopCh,
```

Comment:
do we actually want to loop this fast in prod, or did you just need this for the test? trying to think what would happen if cacher was unable to relist for a period of time (etcd went unavailable or something) and started hotlooping here